GOVERNANCE.md

```diff
@@ -36,30 +41,36 @@ can directly help to shape its future.
 Potential contributors are encouraged to read the [Contributing Guide](http://zarr.readthedocs.io/en/stable/contributing.html)
 as well as the [Code of Conduct](https://github.com/zarr-developers/.github/blob/master/CODE_OF_CONDUCT.md).
 
-A community member becomes a contributor when the following criteria are met:
-
-- At least two core developers from separate employers agree membership is a good idea.
-- The new member has supported the project several times, either through code
-  or otherwise, and has demonstrated a willingness to help solve other people's
-  problems, not just their own.
-- The new member demonstrates some degree of familiarity of open source
-  practices and online courtesy.
```

Comment:
I removed this language because:
- It seems overly heavy-handed and formal
- We are not really following these procedures anyway

Reply:
This is essentially for addition to https://github.com/orgs/zarr-developers/teams/core-devs so I'd say we need _something_. The first bullet point is what we do on https://github.com/orgs/zarr-developers/teams/core-devs/discussions, but happy to modify. The second and third I agree are pretty heavy-handed but typically some recent PR has been pointed at to say, "see, s/he contributed well".

Reply:
Ah, reading below, yeah, I think these bullets are in the wrong place and should be moved down (and additionally simplified).